user/dao/dao.go

```diff
@@ -200,3 +208,21 @@ func (dao *DAO) ReadPicture(input ReadPictureInput) (*Picture, error) {
 
 	return &picture, nil
 }
+
+// UpdatePicture updates a picture in the datastore, returning an error if it fails
+func (dao *DAO) UpdatePicture(input UpdatePictureInput) (*Picture, error) {
+	row := executeQueryWithRowResponse(dao.DB, "UPDATE picture set img = $1 WHERE id = $2 AND user_id = $3 RETURNING *", input.Img, input.ID, input.UserID)
```

Comment:
this is also gonna be a pain in the bum to generate as the SQL query is different to anything else we're currently doing, so will have to be rewritten to select based on multiple conditions 🥇